src/node_constructor.jl

```diff
@@ -781,31 +906,34 @@ function CheckParameters(
             load["pf"] = 0
         elseif load["impedance"] == "RL"
             load["Z"] = 1im * parameters["grid"]["f_grid"] * 2 * pi * load["R"] *
-                load["L"] / (load["R"] + 1im * parameters["grid"]["f_grid"] *
-                2 * pi * load["L"])
+                        load["L"] / (load["R"] + 1im * parameters["grid"]["f_grid"] *
+                                                 2 * pi * load["L"])
             load["pf"] = sign(imag(load["Z"]))abs(cos(atan(imag(load["Z"]) /
-                real(load["Z"]))))
+                                                           real(load["Z"]))))
         elseif load["impedance"] == "RC"
             load["Z"] = load["R"] / (1 + 1im * parameters["grid"]["f_grid"] *
-                2 * pi * load["C"] * load["R"])
+                                         2 * pi * load["C"] * load["R"])
             load["pf"] = sign(imag(load["Z"]))abs(cos(atan(imag(load["Z"]) /
-                real(load["Z"]))))
+                                                           real(load["Z"]))))
         elseif load["impedance"] == "LC"
             load["Z"] = 1im * parameters["grid"]["f_grid"] * 2 * pi * load["L"] /
-                (1 - (parameters["grid"]["f_grid"] * 2 * pi)^2 * load["L"] * load["C"])
+                        (1 - (parameters["grid"]["f_grid"] * 2 * pi)^2 * load["L"] * load["C"])
             load["pf"] = 0
         elseif load["impedance"] == "RLC"
             load["Z"] = 1im * parameters["grid"]["f_grid"] * 2 * pi * load["L"] /
-                (1 + 1im * parameters["grid"]["f_grid"] * 2 * pi * load["L"] / load["R"] -
-                (parameters["grid"]["f_grid"] * 2 * pi)^2 * load["L"] * load["C"])
+                        (1 + 1im * parameters["grid"]["f_grid"] * 2 * pi * load["L"] / load["R"] -
+                         (parameters["grid"]["f_grid"] * 2 * pi)^2 * load["L"] * load["C"])
             load["pf"] = sign(imag(load["Z"]))abs(cos(atan(imag(load["Z"]) /
-                real(load["Z"]))))
+                                                           real(load["Z"]))))
         end
 
-        load["pwr"] = parameters["grid"]["v_rms"]^2 / abs(load["Z"]) *
-            parameters["grid"]["phase"]
+        if !haskey(load, "pwr")
+            load["pwr"] = parameters["grid"]["v_rms"]^2 / abs(load["Z"]) * parameters["grid"]["phase"]
+        end
 
-        load["i_limit"] = sqrt(2)*0.9*parameters["grid"]["v_rms"] / abs(load["Z"])
+        if !haskey(load, "pwr")
```

Comment:
This should be !haskey(load, "i_limint"), correct?

Reply:
Yes!